### DISCUSSION TITLE
ASR识别效果有哪些优化手段

### DISCUSSION THREAD

**Discussion by dabaoyasu** [PaddlePaddle/PaddleSpeech#1458] [Q&A] [answered]
请问：
使用 example/aishell/asr0/ 训练出来的模型，对于识别不准确的内容，可以通过哪些方法进行优化以提升准确率？

**Comment by zh794390558** [comment, marked as answer]
增加出错的数据训练模型。其它工程方法解决。

**Reply by 907122187** [reply]
你好，能麻烦提供一下在conformer_wenetspeech模型的基础上，自己增加出错数据的训练的教程？官网上我自己没有找到

**Reply by zh794390558** [reply]
可以尝试自行构造数据集，然后参考 https://github.com/PaddlePaddle/PaddleSpeech/discussions/1972 做fintune训练。
